problems/euler/Smallest multiple/Java/Farismhg/Multiple.java

```diff
@@ -0,0 +1,32 @@
+public class Multiple
+{
+    // Euclidian algorithm for finding greatest common divisor
+    private static int GCD(int a, int b)
```

Comment:
could maybe use comments on the working of the algorithm, not sure if necessary.